graphql_compiler/__init__.py

```diff
@@ -132,3 +141,36 @@ def graphql_to_gremlin(schema, graphql_query, parameters, type_equivalence_hints
         schema, graphql_query, type_equivalence_hints=type_equivalence_hints)
     return compilation_result._replace(
         query=insert_arguments_into_query(compilation_result, parameters))
+
+
+def get_graphql_schema_from_orientdb_records(schema_records, class_to_field_type_overrides=None,
+                                             hidden_classes=None):
+    """Construct a GraphQL schema from a list of OrientDB schema records.
+
+    Args:
+        schema_records: list of pyorient.otypes.OrientRecord describing the OrientDB schema.
+                        Generated by running ORIENTDB_SCHEMA_RECORDS_QUERY on OrientDB.
+        class_to_field_type_overrides: optional dict, class name -> {field name -> field type},
+            (string -> {string -> GraphQLType}). Used to override the type of a field in the class
```

Comment:
Please make your docstrings consistent with those of the surrounding code. In this one docstring, the `Generated` is indented to under the `list` (i.e. visual indent after the `:`), whereas the second line of the `class_to_field_type_overrides` description is given only a single four-space indent. Especially for a file that will be commonly read by users such as the root `__init__.py` file, we have to make sure our style is consistent and reader-friendly.